registry/handlers/mail.go

```diff
@@ -0,0 +1,45 @@
+package handlers
+
+import (
+	"errors"
+	"net/smtp"
+	"strings"
+)
+
+// mailer provides fields of email configuration for sending.
+type mailer struct {
+	Addr, Username, Password, From string
+	Insecure                       bool
+	To                             []string
+}
+
+// sendMail allows users to send email, only if mail parameters is configured correctly.
+func (mail *mailer) sendMail(subject, message string) error {
+	addr := strings.Split(mail.Addr, ":")
+	if len(addr) != 2 {
+		errors.New("Invalid Mail Address")
```

Comment:
@noxiouz 
I add
```go
return errors.New("Invalid Mail Address")
```
However, I don't understand what kind of check you want in `configureLogHook `.
it just set parameters from configuration. and check has be done in configuration.